tests/cmds/test_legal_hold.py

```diff
@@ -169,6 +173,75 @@
     ]
 }
 """
+EVENTS_RESPONSE = """
```

Comment:
Something seems off about this response.... It is not valid JSON. Should these objects be under a list with key: `legalHoldEvents`

Reply:
Or else I believe line 290 will create a response where this a `str` type (I think).

Reply:
I don't think the fixture that uses this is ever used, if that is the, case can this be removed?

Reply:
Yes, that is no longer needed, since I moved to a generator format for the mocker. I forgot to take it out on my last push along with it's corresponding fixture.